go.sum

```diff
@@ -401,6 +401,8 @@ github.com/yuin/goldmark v1.3.5/go.mod h1:mwnBkeHKe2W/ZEtQ+71ViKU8L12m81fl3OWwC1
 github.com/yuin/goldmark v1.4.13/go.mod h1:6yULJ656Px+3vBD8DxQVa3kxgyrAnzto9xy5taEt/CY=
 go.mongodb.org/atlas v0.23.2-0.20230323121922-76f1516f3ff9 h1:5/GxsRQFvAW9UZD/GAcjQMQAfEWTjvdv1INYjqoL53w=
 go.mongodb.org/atlas v0.23.2-0.20230323121922-76f1516f3ff9/go.mod h1:r5HHjwxout7CJwjkAcJt+c2yA43rrB6LXidPk3pyeuo=
+go.mongodb.org/atlas v0.23.2-0.20230328111225-e40463c98c4e h1:Uy+DI9phtg3/M28DYH6u8Qvhh033yUhYlWlUqI2x2Iw=
```

Comment:
can you run `go mod tidy` when updating the client?

Reply:
Done. Thank You!